app/webserver/handler.go

```diff
@@ -260,7 +261,33 @@ func uploadSendAttachment(attachment UploadAttachmentMessage) error {
 		log.Errorln("[axolotl] attachment error: Attachment too large, not sending")
 		return nil
 	}
-	err, m := sender.SendMessageHelper(attachment.To, attachment.Message, file, nil)
+	m, err := sender.SendMessageHelper(attachment.To, attachment.Message, file, nil, false)
+	if err == nil {
+		go MessageHandler(m)
+	}
+	return nil
+}
+func uploadSendVoiceNote(voiceNote SendVoiceNoteMessage) error {
+	log.Debug("[axolotl] uploadSendVoiceNote to ", voiceNote.To)
+	// Do not allow sending attachments larger than 100M for now
+	var maxAttachmentSize int64 = 100 * 1024 * 1024
```

Comment:
I never thought about consts in functions in go. I think this can be improved in many places.

Reply:
What I meant was defining the max attachment size below the imports as
```golang
// Do not allow sending attachments larger than 100M for now
const MAX_ATTACHMENT_SIZE int64 = 100 * 1024 * 1024
```
because then its only defined once and easily found.

Reply:
Thank you, i never thought about consts in go functions but you are right.

Reply:
The go linter doesn't like upper case consts so it needs to be MaxAttachmentSize and i put it in the config submodule.